src/@next/TextArea/TextAreaStyle.ts

```diff
@@ -0,0 +1,98 @@
+import styled from 'styled-components';
+import { borderRadius4 } from '../utilities/borderRadius';
+import { Neutral, Red } from '../utilities/colors';
+import { NotoSans } from '../utilities/fonts';
+
+interface TextAreaProp {
+  width: string;
+}
+
+export const StyledTextAreaContainer = styled.div<TextAreaProp>`
+  position: relative;
+  width: ${props => props.width};
+
+  border: 1px solid ${Neutral.B68};
+  border-radius: ${borderRadius4};
+  padding: 8px 12px 25px 12px;
+  cursor: text;
+
+  &:not([data-has-counter='true']) {
+    padding-bottom: 8px;
+  }
+
+  &[data-focus='true'] {
+    outline: none;
+    box-shadow: 0px 0px 0px 1px ${Neutral.B100}, 0px 0px 0px 3px #6ac9ec;
+  }
+
+  &[data-error='true'] {
+    border: 1px solid ${Red.B93};
+
+    &[data-focus='true'] {
+      box-shadow: none;
+    }
+  }
+
+  &[data-disabled='true'] {
+    border: 1px solid ${Neutral.B85};
+    background: ${Neutral.B95};
+  }
+`;
+
+export const StyledTextArea = styled.textarea<TextAreaProp>`
+  background: ${Neutral.B100};
+  box-sizing: border-box;
+  border: none;
+
+  font-family: ${NotoSans}, sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 150%;
```

Comment:
you can export `body1` from `TypographyStyles.ts` and import it here inline instead to apply a much more accurate styles for this. Just rename the import name so it's easy to know that it's the body1 typography styles applied. it'll be easier to maintain as well since we don't have to bother maintaining this separately in case there are changes with Typography styles in the future.